database/mssql/scripts/dbscripts/PSP_PIMS_S10_00/Alter Down/01_PSP_PIMS_S10_00_Alter.sql

```diff
@@ -0,0 +1,9602 @@
+// Script generated by Aqua Data Studio Schema Synchronization for MS SQL Server 2016 on Fri Sep 03 17:12:13 PDT 2021
```

Comment:
I am assuming the **PSP_PIMS_S10_00** folder contains the scripts for Sprint 10?

Reply:
Yes, Jeremy. The README explains the naming convention